### PR TITLE
Dtspo 1783 creating default http port lookup

### PR DESCRIPTION
**Pull request by SimonGivan**
Changing backend http port to allow alternative values when this module is called.

### CODE REVIEW

File: frontdoor.tf [hunk 1]
```diff
@@ -128,7 +128,7 @@ resource "azurerm_frontdoor" "main" {
         content {
           host_header = lookup(host.value, "host_header", host.value["custom_domain"])
           address     = domain.value
-          http_port   = 80
+          http_port   = lookup(host.value, "http_port", []) == [] ? 80 : host.value["http_port"]
```
Comment (timja) [inline frontdoor.tf:131]:
```suggestion
          http_port   = lookup(host.value, "http_port", 80)
```

**Review by timja — CHANGES_REQUESTED** [review]
I think you overcomplicated it

File: frontdoor.tf (hunk 1 above)
Comment (SimonGivan) [reply frontdoor.tf:131]: It's to allow a different http port to be used when calling this module. Heritage need to use port 81 and not port 80

Comment (timja) [reply frontdoor.tf:131]: yes but your code is wrong, lookup has a default value

Comment (SimonGivan) [reply frontdoor.tf]: Have amended it to include your change

**Review by luigibk — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**